src/bin/pg_dump/pg_dump.c

```diff
@@ -12394,6 +12394,10 @@ dumpFunc(Archive *fout, const FuncInfo *finfo)
 		appendPQExpBufferStr(q,
 							 "SET babelfishpg_tsql.restore_tsql_tabletype = TRUE;\n");
 
+	if (strcmp(lanname, "pltsql") == 0)
+		appendPQExpBufferStr(q,
+							 "SET babelfishpg_tsql.dump_restore = TRUE;\n");
```

Comment:
This may have a side effect for functions/procedures under sys schema because our current assumption is that those functions and procedures are created through a PG connection. What about excluding those?

Reply:
The issue seems to be only with ITVFs, so changed the condition to only set the GUC if it's an ITVF function.